Muvr/MRLabelViewController.swift

```diff
@@ -131,13 +152,18 @@ class MRLabelViewController : UIViewController, UITableViewDelegate, UITableView
             l.intensity = Double(intensity.value) / Double(intensity.maximumValue)
             l.repetitions = repetitions.text.flatMap { UInt32($0) } ?? UInt32(0)
             l.weight = weight.text.flatMap { Double($0) } ?? Double(0)
-                
+            
+            NSLog("start = \(l.start.formatTime())")
+            NSLog("end = \(l.end.formatTime())")
             MRAppDelegate.sharedDelegate().saveContext()
         }
         
-        if sender.tag == 0 {
-            doStart()
-        } else {
+        if timer == nil {
+            timer = NSTimer.scheduledTimerWithTimeInterval(1, target:self, selector: Selector("updateCounter"), userInfo: nil, repeats: true)
+            counter = 5
```

Comment:
I think 3 seconds is enough. Added to the 10% of each exercise we are going to throw away anyway this will leave us with a window of ~ 6 seconds to pick up the weights.